### DISCUSSION TITLE
How to upload logo?

### DISCUSSION THREAD

**Discussion by jonaskohl** [TandoorRecipes/recipes#3124] [Q&A]
***Nevermind, I just found it. I feel stupid...***

<del>This may be a stupid question, but I just installed a new instance of Tandoor (via Docker) and now I'm trying to change the logo in the navigation bar. But when I click the dropdown on the Space Management page, it just says "List is empty". How do I add a logo file to this list?</del>

**Comment by smilerz** [comment]
`/list/user-file/`

**Reply by CaldwellYSR** [reply]
What does this mean? Is this meant to be in `/mediafiles` is this meant to be in the root directory with a volume?

**Comment by CaldwellYSR** [comment]
For anyone coming to this from google later... it's `https://your-url/list/user-file` in case that isn't obvious from above.

**Comment by hatty24** [comment]
Sorry, but I still don't understand how or where I can place my logo,
to be able to select it in the list.

**Comment by hatty24** [comment]
Okay, I've found it!
Sometimes you think too complicated. 🙄

**Comment by MixamTheGreat** [comment]
OK so maybe I'm missing something but I can't find where to put the logo. Where is _/list/user-file/_?

**Reply by smilerz** [reply]
that's literally the URL path

**Reply by MixamTheGreat** [reply]
OK, I feel so stupid now. I was looking for a /list/userfile/ directory in the docker folder. So navigate to http://tandoorurlwhatever/list/user-file/ and there's an upload form.
